tcms_junit_plugin/__init__.py

```diff
@@ -15,8 +16,9 @@ class Backend(plugin_helpers.Backend):
 
 
 class Plugin:  # pylint: disable=too-few-public-methods
-    def __init__(self, verbose=False):
+    def __init__(self, summary_template, verbose=False):
```

Comment:
This will break inherited classes in case someone is inheriting from this implementation. Better add it to the end of the signature with a default value to make it compatible.